tests/unit/scripting.tcl

```diff
@@ -119,6 +119,10 @@ start_server {tags {"scripting"}} {
         r evalsha fd758d1589d044dd850a6f05d52f2eefd27f033f 1 mykey
     } {myval}
 
+    test {EVALSHA_RO - Can we call a SHA1 if already defined?} {
+        r evalsha_ro fd758d1589d044dd850a6f05d52f2eefd27f033f 1 mykey
+    } {myval}
+
```

Comment:
why did we clone that test?

Reply:
covergae for EVALSH_RO

Reply:
there is one in acl.tcl.
but i see i was wrong, it's not a dup of the above test (eval vs evalsha)